pkg/rpcclient/actor/waiter.go

```diff
@@ -34,37 +45,60 @@ type (
 	// RPCWSWaiter is an interface that enables improved transaction awaiting functionality
 	// for Actor instance based on web-socket Block and ApplicationLog notifications.
 	RPCWSWaiter interface {
-		RPCPollingWaiter
```

Comment:
I think it's OK to include it if that's the way WSWaiter is (falling back to poll-based one if anything goes wrong).